packages/ra-core/src/util/Ready.tsx

```diff
@@ -65,37 +65,41 @@ const Button = ({ img, label, href }) => {
-                alt="react-admin logo"
-            />
-            <h1>Welcome to React-admin</h1>
-            <div>
-                Your application is properly configured.
-                <br />
-                Now you can add a &lt;Resource&gt; as child of &lt;Admin&gt;.
+export default () =>
+    process.env.NODE_ENV === 'production' ? (
+        <span />
+    ) : (
+        <div style={styles.root}>
```

Comment:
It's probably better to use material-ui styles here instead of inline styles.

Reply:
no, it's in ra-core, there is no material-ui here.